projects/base-library/project/Core/Src/main.c

```diff
@@ -119,24 +116,35 @@ int main(void)
   MX_I2C3_Init();
   MX_I2C4_Init();
 
-#ifdef TEST_MODE
+
+/*#ifdef TEST_MODE
   run_tests();
 #endif
+*/
```

Comment:
Why is this commented out? would it not compile?

Make sure the 'Tests' folder in your stm32 project (under Project Explorer tab) is a source folder (has a tiny blue 'C' icon on it)

Reply:
Any update on this?